File: app/src/main/java/com/example/util/timereminder/data/prefs/BasePreferencesHelper.java
package com.example.util.timereminder.data.prefs;

public interface BasePreferencesHelper {

    long getDateOfBirthUTC();

    long getDateOfDeathUTC();

    boolean isSettingsSetUp();

    boolean showMinutes();

    boolean showHours();

    boolean showDays();

    boolean showYears();

}

File: app/src/main/java/com/example/util/timereminder/data/prefs/BasePreferencesHelper.kt
package com.example.util.timereminder.data.prefs

interface BasePreferencesHelper {

    fun getDateOfBirthUTC(): Long

    fun getDateOfDeathUTC(): Long

    fun isSettingsSetUp(): Boolean

    fun showMinutes(): Boolean

    fun showHours(): Boolean

    fun showDays(): Boolean

    fun showYears(): Boolean

}
